### DISCUSSION TITLE
ISO Certification

### DISCUSSION THREAD

**Discussion by MaxSchilling** [nhost/nhost#10] [Ideas]
To improve credibility of  security and quality: ISO Certifications would help. Eg. ISO9001 (QM) and ISO27001 /SOC 1,2,3
See firebase (there is a table) https://firebase.google.com/support/privacy 
Customers with sensitive data might get audited (like us) and it helps A LOT to justify the use of Nhost, if you are already certified. Especially, if people look for secure GDPR compliant solutions, they might come from a sensitive data field of business where those certifications are standard.
Supporting pen tests from your side might come up as well, as some instutions want proof (e.g. Healtchcare sector).

Keep up the good work!
Hope to be a nhost user of the future.

Regards,
Max

**Comment by nunopato** [comment]
@MaxSchilling thanks for bringing this up! this is something we plan to work on this year, after some restructuring of our current infrastructure.
